tests/test1.csv

```diff
@@ -0,0 +1 @@
+one,two,three
```

Comment:
Please use better names for this. Maybe fake tickers?

Reply:
done